test/io.jl

```diff
@@ -33,7 +33,7 @@
             @test h.ispg === Int32(4)
             @test h.nsymbt == sizeof(eh) == Int32(160)
             @test h.extra1 === Tuple(zeros(UInt8, 8))
-            @test h.exttyp === ""
+            @test h.exttyp === "\0\0\0\0"
```

Comment:
Where does this change come from?

Reply:
I changed `function bytestoentry(name, type, pointer)` in `header.jl` to make it consistent with mrcfile.